substratevm/src/com.oracle.graal.pointsto.standalone/src/com/oracle/graal/pointsto/standalone/PointsToAnalyzer.java

```diff
@@ -196,6 +201,28 @@ private PointsToAnalyzer(String mainEntryClass, OptionValues options) {
         }
     }
 
+    private String getAnalysisName(String entryClass, OptionValues options) {
+        String entryPointsFile = StandaloneOptions.AnalysisEntryPointFile.getValue(options);
+        String entryPointsFileOptionName = StandaloneOptions.AnalysisEntryPointFile.getName();
+        mainEntryIsSet = entryClass != null && !entryClass.isBlank();
+        entrypointsAreSet = entryPointsFile != null && entryPointsFile.length() != 0;
+        if (!mainEntryIsSet && !entrypointsAreSet) {
+            AnalysisError.shouldNotReachHere(
+                            "No analysis entry are specified. Must set entry class or -H:" + entryPointsFileOptionName + " to specify the analysis entries.");
```

Comment:
So you now allow to specify all entry points via the file and the `AnalysisEntryClass` can be empty. Please update the options help to reflect that.

Reply:
Fixed